### PR TITLE
Update certificates_tab.md

### PR DESCRIPTION
**Pull request by BryanGower**

### CODE REVIEW

File: certificates_tab.md [hunk 1]
```diff
@@ -30,10 +29,10 @@ The main source of data is the RECs table. In the table you will notice the foll
 -   Fuel Type: Solar, Wind, Hydro, etc.
 -   Vintage: When the RECs were created
 -   Location: The location of your generator
--   Type: The type of certificate, most likely REC
 -   Eligibility: This column will display for which programs these RECs are eligible
 -   Quantity: The number of RECs issued
--   RRC Adjusted Quantity:  This is specific to the Wisconsin RRC program. Not sure what the Wisconsin RRC program is? Then this most likely can be ignored.
+-   Serial Numbers
+-   Details: Information on the REC batc
```
Comment (SeanDarling) [inline certificates_tab.md:35]: typo: batc

**Comment by BryanGower** [comment]
@SeanDarling I updated the typo. Thanks for catching it.

**Review by markmrets — COMMENTED** [review]
all good

**Review by markmrets — COMMENTED** [review]
ALL GOOD

File: certificates_tab.md
Comment (Queerty) [inline certificates_tab.md:1]: For line 5 we will want to only include the "Please note.." once, as its repeated in plain text and italicized. 

Please note, that your tabs on the left may vary. ***Please note, your tabs on the left may vary.***

Comment (Queerty) [reply certificates_tab.md:1]: I would also offer a little bit of context to the gear icon image (i.e. located at the top right of the table) if we are going to just include the image of it without any of the dashboard visible.

**Review by Queerty — APPROVED** [review]
Your RECs view should look something like this. Please note, that your tabs on the left may vary. ***Please note, your tabs on the left may vary.***

For this

**Pull request closed without merging**